LICENSE

```diff
@@ -0,0 +1,54 @@
+BSD-2-Clause Plus Patent License
+
+Copyright 2024 Intel Corporation.
```

Comment:
should be 2022 - 2024 (first docs were added to this repo 2022...)

Reply:
Done.